### PR TITLE
moving images/reformatting buttons/Adding missing CMYK images for grey icon

### PR DESCRIPTION
**Pull request by rickiesherman**
Wrapping up the content editing in styleguide. Thanks to Taylor and Tony for helping me :)

### CODE REVIEW

File: src/html/runnabear.html [hunk 1]
```diff
@@ -7,105 +7,104 @@ <h2 class="h2 weight-strong">Full Image</h2>
...
       <img class="img" src="images/bears/doctor-bear.png" width="180" alt="Doctor Bear">
       <figcaption class="figcaption">Doctor Bear</figcaption>
-      <a class="btn btn-xs white" href="images/bears/doctor-bear.png" download>Download</a>
+      <a class="btn btn-xs white" href="images/bears/RGB/doctor-bear.png">Download PNG</a>
     </figure>
     <figure class="grid-block figure vertical shrink align-center justify-right figure-bears">
       <img class="img" src="images/bears/error-bear.png" width="180" alt="Error Bear">
       <figcaption class="figcaption">Error Bear</figcaption>
-      <a class="btn btn-xs white" href="images/bears/error-bear.png" download>Download</a>
+      <a class="btn btn-xs white" href="images/bears/RGB/error-bear.png" Download PNG>Download PNG</a>
```
Comment (runnabro) [inline src/html/runnabear.html]: Shouldn't have PNG in the attribute

File: src/images/logo/gray-icon/CMYK/gray-icon.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (taylordolan) [inline src/images/logo/gray-icon/CMYK/gray-icon.svg]: This should be minified.

File: src/html/runnabear.html [hunk 3]
```diff
@@ -7,105 +7,104 @@ <h2 class="h2 weight-strong">Full Image</h2>
     <figure class="grid-block figure vertical shrink align-center justify-right figure-bears">
       <img class="img" src="images/bears/basic-bear.png" width="180" alt="Basic Bear">
       <figcaption class="figcaption">Basic Bear</figcaption>
-      <a class="btn btn-xs white" href="images/bears/basic-bear.png" download>Download</a>
-    </figure>
+      <a class="btn btn-xs white" href="images/bears/RGB/basic-bear.png" Download>Download PNG</a>
```
Comment (taylordolan) [inline src/html/runnabear.html]: Looks like this first figure is missing its closing tag:
image

**Review by taylordolan — CHANGES_REQUESTED** [review]

File: src/html/runnabear.html (hunk 3 above)
Comment (rickiesherman) [reply src/html/runnabear.html]: oop, got it

File: src/images/logo/gray-icon/CMYK/gray-icon.svg (hunk 2 above)
Comment (rickiesherman) [reply src/images/logo/gray-icon/CMYK/gray-icon.svg]: 👍

**Review by taylordolan — APPROVED** [review]

File: src/html/logo.html [hunk 4]
```diff
@@ -4,7 +4,7 @@ <h1 class="h1 weight-strong">Logo</h1>
   <h2 class="h2 weight-strong">Icon</h2>
   <div class="figure-wrapper">
     <figure class="grid-block figure vertical shrink figure-logo-icon">
-      <img class="img" src="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.svg" width="190" height="190" alt="Lockup Purple">
+      <img class="img" src="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.svg" width="292" alt="Lockup Purple">
```
Comment (taylordolan) [inline src/html/logo.html]: All images should have both `width` and `height` attributes. Let's go ahead and add any attributes that are missing to the logos/icons and bears.

**Review by taylordolan — CHANGES_REQUESTED** [review]

File: src/html/logo.html [hunk 5]
```diff
@@ -1,10 +1,10 @@
-<!-- section -->
+  <!-- section -->
```
Comment (runnabro) [inline src/html/logo.html]: Unnecessary indent?

**Pull request merged**